scripts/paper/aggregate_stwo_phase71_handoff_receipt_benchmark.py

```
@@ -0,0 +1,178 @@
+            benchmark_version = payload["benchmark_version"]
+            semantic_scope = payload["semantic_scope"]
+            timing_unit = payload.get("timing_unit", "milliseconds")
+            canonical_rows = payload["rows"]
+            build_row_map(canonical_rows, source=input_path)
+            for row in canonical_rows:
+                timing_samples[key_for(row)] = []
+        else:
+            if payload["benchmark_version"] != benchmark_version:
+                raise SystemExit(
+                    f"benchmark_version mismatch in {input_path}: {payload['benchmark_version']} != {benchmark_version}"
+                )
```

Ruff (0.15.11)

[warning] Avoid specifying long messages outside the exception class

(TRY003)